plugins/inputs/pgbouncer/pgbouncer.go

```diff
@@ -0,0 +1,196 @@
+package pgbouncer
+
+import (
+	"bytes"
+	"database/sql"
+	"regexp"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	"github.com/lib/pq"
```

Comment:
if you're ignoring columns due to lib/pq's poor handling of certain datatypes, you might want to look at pgx (used by the postgresql plugins) it has better support for native psql types.

Reply:
Any news on this, guys?

Reply:
https://github.com/influxdata/telegraf/pull/2819 was preferred to this one
I suspect that this one can be closed

Reply:
I think you are right that we can close this, but if we had an example configuration for postgresql_extensible that does the queries here we could add it to the docs and link to it from the main README.

I'd happily accept a PR with this or just comment with a tested example and I will add it.